stonesoup/models/measurement/nonlinear.py

```diff
@@ -2,6 +2,7 @@
 import copy
 from typing import Sequence, Tuple, Union
 
+from math import sqrt
```

Comment:
Could `np.sqrt` be used instead?

Reply:
I generally find that math.sqrt is appreciably faster for scalar operations than Numpy. But happy to change to Numpy if desired.
[https://shocksolution.com/2009/01/09/optimizing-python-code-for-fast-math/](url)

Reply:
I can confirm; running some brief tests locally with this Jacobian update suggests the `math.sqrt` implementation is slightly faster (~1%) than `np.sqrt` on arm64.